src/main/java/net/fabricmc/loader/api/EntrypointException.java

```diff
@@ -17,15 +17,14 @@
 package net.fabricmc.loader.api;
 
 public class EntrypointException extends RuntimeException {
-	public EntrypointException(Throwable t) {
-		super(t);
-	}
+	private final String key;
 
-	public EntrypointException(String s) {
-		super(s);
```

Comment:
I think pruning default exception constructors is unnecessary for this PR.

Reply:
`EntrypointException` means an exception inside an entrypoint, so it must have a cause.

Reply:
The class isn't meant to be constructed by mods, maybe we can clarify this explicitly?